src/js/components/accordion.js

```diff
@@ -53,6 +54,33 @@ const toggleButton = (button, expanded) => {
   }
 };
 
+/**
+ * Toggle the Collapse/Expand all text label.
+ * @param {HTMLElement} button
+ */
+const toggleText = (element) => {
+  let controlBtn = element;
+
+  const originalString = controlBtn.textContent.trim();
+  const startText = controlBtn.getAttribute("data-label--expanded");
+  const endText = controlBtn.getAttribute("data-label--collapsed");
+
+  controlBtn.textContent = (originalString === startText) ? endText : startText;
```

Comment:
Can you just use the expanded attribute to set text content based? Or do we need the data-label attribute? This check seems unnecessary.

Reply:
For the original string? I felt like it was more obvious this way what texts are being toggled. Happy to revisit.